src/samtools/samtools_sort/config.vsh.yaml

```
@@ -0,0 +1,149 @@
+name: samtools_sort
+namespace: samtools
+description: Sort SAM/BAM/CRAM file.
+keywords: [sort, bam, sam, cram]
+links:
+  homepage: https://www.htslib.org/
+  documentation: https://www.htslib.org/doc/samtools-idxstats.html
+  repository: https://github.com/samtools/samtools
+references: 
+  doi: [10.1093/bioinformatics/btp352, 10.1093/gigascience/giab008]
+license: MIT/Expat
+
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --input
+        type: file
+        description: SAM/BAM/CRAM input file.
+        required: true
+        must_exist: true
+  - name: Outputs
+    arguments:
+      - name: --output
+        type: file
+        description: |
+          Write final output to file.
+        required: true
+        must_exist: false
```

Output should always be written to file, not to stdout

```suggestion
```

---

Hey, isn't this how we make sure it's written to a file but without requiring it to exist before running to command? Without this line we need to have an empty file beforehand to which we write the output. This is how I did it in previous components. Please let me know! thanks

---

Correction: the direction should be set to `output`

```suggestion
        direction: output
```